packages/lab/stories/input-next/input-next.doc.stories.mdx

```diff
@@ -0,0 +1,109 @@
+import { InputNext } from "@salt-ds/lab";
+import { Meta, Canvas, ArgsTable } from "@storybook/addon-docs";
+import {
+  CSSClassTable,
+  CharacteristicUsage,
+} from "css-variable-docgen-components";
+import { Link } from "@salt-ds/core";
+import HelpAndSupport from "docs/blocks/help-and-support.doc.mdx";
+
+<Meta
+  title="Documentation/Lab/Input Next"
+  component={InputNext}
+  parameters={{
+    viewMode: "docs",
+  }}
+/>
+
+# Input Next
+
+Input provides an editable field in which users can enter text and numeric values.
+
+# Examples
+
+## Default
+
+Input comprises value text and a focus ring. The component is initially empty and then, once it has focus, it accepts text on a single line.
+
+<Canvas>
+  <Story id="lab-input-next--default" />
+</Canvas>
+
+## Controlled
+
+You can control Input with the `value` and `onChange` props. Learn more about <Link href="https://react.dev/learn/sharing-state-between-components#controlled-and-uncontrolled-components">controlled vs. uncontrolled components in the official React docs</Link>.
```

Comment:
Minor: I would expect the Link to wrap a different part of this sentence but maybe worth checking?

e.g.

```suggestion
You can control Input with the `value` and `onChange` props. Learn more about <Link href="https://react.dev/learn/sharing-state-between-components#controlled-and-uncontrolled-components">controlled vs. uncontrolled components</Link> in the official React docs.
```

```suggestion
You can control Input with the `value` and `onChange` props. Learn more about controlled vs. uncontrolled components in the <Link href="https://react.dev/learn/sharing-state-between-components#controlled-and-uncontrolled-components">official React docs</Link>.
```